ark/attest/cache/ts.ts

```diff
@@ -99,13 +100,14 @@ export type TsconfigInfo = {
 }
 
 export const getTsConfigInfoOrThrow = (): TsconfigInfo => {
-	const configFilePath = ts.findConfigFile(
-		fromCwd(),
-		ts.sys.fileExists,
-		"tsconfig.json"
-	)
+	const config = getConfig()
+	const configFilePath =
+		config.tsconfig ??
+		ts.findConfigFile(fromCwd(), ts.sys.fileExists, "tsconfig.json")
```

Comment:
I wasn't sure we wanted this fallback here, since the default config should already have attempted to reach `tsconfig.json` (although without using `ts.findConfigFile`), or if maybe we want to keep `config.tsconfig` undefined so that the path is resolved here.